static-files/ocp_on_premise.yaml

```diff
@@ -0,0 +1,178 @@
+---
+generators:
+  - OCPGenerator:
+      start_date: 2019-05-01
+      end_date: 2019-07-31
+      nodes:
+        - node:
+          node_name: compute_1
```

Comment:
I started with 3 compute nodes and 3 master nodes with the cpu/memory that a default OpenShift cluster would spin up in AWS.